pages/villageHQ.php

```diff
@@ -275,16 +277,16 @@ function villageHQ() {
 		if (isset($_POST['submit_exam'])) {
 			try {
 				// check if already sensei
-				if (SenseiManager::isSensei($player->sensei_id, $system)) {
-                    throw new RuntimeException('You do not meet the requirements!');
+				if (SenseiManager::isActiveSensei($player->sensei_id, $system)) {
+                    throw new Exception('You do not meet the requirements!');
```

Comment:
More merge conflicts here
```suggestion
                    throw new RuntimeException('You do not meet the requirements!');
```